### ISSUE TITLE
Unable to Run - GPTH Quitted 

### ISSUE THREAD

**Issue by nothingshocking** [TheLastGimbus/GooglePhotosTakeoutHelper#318]
I have been unable to successfully run GPTH on my windows machine. I've followed the instructions and tried different combinations to determine if there was something wrong on my end. 

I exported 3 Google Photos folders using Takeout. I extracted those zip files and placed the resulting folders under a Takeout folder as described in the instructions. 
![image](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/68715295/de7bd00f-2138-4877-8435-5d6c7432b428)
![image](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/68715295/9a0fc172-83b5-4057-81ac-7a3239f85c28)
![image](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/68715295/21e752fb-4e1a-4794-b2de-e18586267b60)

When running GPTH I used C:/Downloads/Takeout as the input folder and C:/Downloads/GPTH Output as the output folder. I chose 1 and 2 for the options presented and get the following error. 
![image](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/68715295/b08e3d31-93c2-4ef6-821c-f2d4c5cfc1be)

I've tried the same but having all the "year" folders under one Takeout folder with the same results. I've read some of the closed issues related to Code 13 but the answers there were basically follow the instructions. I'm not sure why it isn't working for me. Any advice would be much appreciated.

**Comment by nothingshocking** [comment]
Any thoughts... Bueller?

**Comment by palijn** [comment]
You wrote you used `C:/Downloads/Takeout`as input folder, but the folder where your Takeout is in the screenshot is `C:/Downloads/Takeout/takeout2024...01/Takeout`  . Try this ?

**Comment by TheLastGimbus** [comment]
You have a wrong folder structure, look at the visual guide more carefully

**Comment by nothingshocking** [comment]
I am sorry, I must be slow. Your guide doesn't actually explain or show what the file structure should look like. It just says to copy all the Takeout folders into one.

When extracting one of the zip files I get the following structure:
takeout-20240610T183930Z-001.zip >>  takeout-20240610T183930Z-001/Takeout/Google Photos/_variousfolders_

Your visual guide indicates copying each of the takeout-20240610T183930Z-00X folders into a main Takeout folder which would result in what I showed above. 
C:/Downloads/Takeout/takeout-20240610T183930Z-00x/Takeout/Google Photos/_variousfolders_

Why not just show/explain the final folder structure in the write up? I have looked at it carefully, multiple times.

**Comment by nothingshocking** [comment]
> You wrote you used `C:/Downloads/Takeout`as input folder, but the folder where your Takeout is in the screenshot is `C:/Downloads/Takeout/takeout2024...01/Takeout` . Try this ?

Try what?

**Comment by palijn** [comment]
try using the second path as gpth input folder. 

Le 5 juillet 2024 16:48:03 GMT+02:00, nothingshocking ***@***.***> a écrit :

**Comment by nothingshocking** [comment]
I've tried every combination of structure I can think of and I am still getting Code 13. 

I have 7 zip files for the photos from a few different years. The instructions says "Copy all 'Takeout' folders into one" which resulted in the structure I shared in my first post. Based Gimbus' response today I took all of the "year" folders from the various extract folder structures and combined them into one set of "year" folders under on C:/Downloads/Takeout/Google Photos/ folder.  
![image](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/68715295/31c9c80d-0426-441f-8beb-9561c2daf49a)

I ran it using C:/Downloads/Takeout as the source - Code 13
I ran it using C:/Downloads/Takeout/Google Photos as the source - Code 13
I removed Google Photos from the structure and had C:/Downloads/Takeout/_yearfolders_  ran it using C:/Downloads/Takeout as the source - Code 13

@TheLastGimbus My question stands, what is the expected file structure suppose to be if not one of the combinations I've tried?

**Comment by palijn** [comment]
@TheLastGimbus the last folder structure looks legit, so, helping out seems to be on you now, sorry.

**Comment by nothingshocking** [comment]
@TheLastGimbus I found the issue. The Takeout zip files I received had the years folder as shown above (2013, 2014, 2015). The many times I tried to run the tool using C:/Downloads/Takeout as the source, or variations on the folders as the source I kept getting Code 13. In reading through other issue threads I saw some people referenced the year folders being labeled **"Photos from XXXX"**. I renamed my folders accordingly and used C:/Downloads/Takeout as the source and it worked. 

![image](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/68715295/9531a855-0449-46da-bdf0-cbb64331dd91)

I know you put a lot of time and energy into this and I like others am very appreciative. At the same time, like any piece of code the instructions can have bugs or areas for optimization. I notice in the issues there are numerous variations of the same question I had. What is the correct source folder structure? Your visual instructions are nice but they don't directly explain the expected / required folder structure. It also doesn't say the year folders should be named in a specific format. The instruction could use some optimization to include a bit more detail.

**Comment by TheLastGimbus** [comment]
It's very weird that your takeout didn't have "Photos from..." folders - literally everyone's elese do have them. Did you select only albums while takeout-ing, or have the takeout from the past (there used to be other format for this)

Also, which country you're from? This may also affect -_-

tl;dr noone else had a problem like yours, that's why readme doesn't cover this

**Comment by Ark-uki** [comment]
I am chinese and the takeout was named in Chinese by default. I faced the same exit code 13, then I read this issue and renamed all folders in English. The program ran well then. So different country might cause different structures.

**Comment by ghost** [comment]
Hey, same here when the extract is in french, I get takeouts folders "Photos de 20xx". Changing the names manually did the trick.

**Comment by germandouce** [comment]
Hello! Same issue here. When the extract is in spanish, I get takeout folders "Fotos de 20xx".

Changing the names manually in the .zip's to "Photos from 20xx" solve it.

I would suggest adding some kind of Warning in the readme for non english speakers (who's account will probably not be in english)

**Comment by lhofmanngmx** [comment]
Hello,
indeed, I also can confirm this issue and solution from German language, and Linux Mint perspective. As Google extracts the relevant folder name to "Fotos von 2024", by my browser or my Google account. As soon I change just the folder name to "Photos from 2024", everything works fine. I did this _translation_ just after extraction of the zip-files. 
For me it works, so I can use the tool .

Thanks a lot to the developers.

Ludwig
